### DISCUSSION TITLE
Colorize new titles in the articles list

### DISCUSSION THREAD

**Discussion by scottyme** [martinrotter/rssguard#1460] [General]
I love the blue colorization of the feeds with new articles in the feeds list. Can new titles in the articles list be colorized also?

**Comment by martinrotter** [comment]
Hi.

RSS Guard does not store "new" flag in articles because "new" can mean different things for different users.

But, there is a "Highlight" feature (see article list toolbar), where you can mark either unread or important (or both) articles with the same color as new feeds.

![image](https://github.com/user-attachments/assets/80af9eda-d9b8-49ee-acf5-468dfe26a849)

**Comment by scottyme** [comment]
Thanks for the quick response. I know you're busy so I appreciate it.
What criteria does RSS Guard used to color the feeds list? 
Can I highlight titles fetched after a certain time--say, when the application is launched, or when fetch is invoked? If not, can I accomplish what I want with Article Filtering? 
I just need a way to quickly see articles that have been added by the latest fetch.

**Reply by martinrotter** [reply]
I see. Well. This feature is currently not really there. I am not sure about some workaround.

Would have to be added as a feature. I will create a ticket from this discussion.

**Reply by martinrotter** [reply]
https://github.com/martinrotter/rssguard/issues/1461

**Reply by scottyme** [reply]
> Would have to be added as a feature. I will create a ticket from this discussion.

That's fantastic! Thank you!
